react-stopwatch-component/dist/styles.css

```diff
@@ -0,0 +1,16 @@
+.circle {
+  width: 100px;
+  height: 100px;
+  border-radius: 50%;
+  border: 1px solid black
+}
+
+.fa-play, .fa-pause {
+  position: relative;
+  left: 2.7rem
+}
```

Comment:
Avoid styling classes from external libraries. Also, don't use positioning to center things.